examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -39,6 +39,8 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.    get_booking(input_data_folder)
</pre><p>Please do not change this cell and do not assign path to <code>input_data_folder</code>  bcs the test is failing. you dont need to change here. Please use the original script. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/975/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>